go/private/tools.bzl

```diff
@@ -0,0 +1,69 @@
+# Copyright 2014 The Bazel Authors. All rights reserved.
```

Comment:
Not all things would be tests.
I was going to stick all the tools in one file, but how about this structure, a tools directory with a bzl file per tool, so go/private/tools/vet.bzl